src/api/app/views/webui/request/beta_show_tabs/_conversation.html.haml

```diff
@@ -32,6 +32,20 @@
         - staging_project = review.project
         Is staged in
         = link_to(review.by_project, staging_workflow_staging_project_path(staging_project.staging_workflow.project, staging_project.name))
+
+    - if bs_request.accept_at.present?
+      %h4.mt-3 Request Auto-Acceptance
+      .pl-3
+        - if BsRequest::FINAL_REQUEST_STATES.include?(bs_request.state)
+          At
+          %span.fuzzy-time{ title: "#{l bs_request.accept_at}" }
```

Comment:
Could we prefix the `l` method with `I18n.`, so it's a bit clearer where this is coming from. This would result in:

```suggestion
          %span.fuzzy-time{ title: "#{I18n.l bs_request.accept_at}" }
```

This also applies to other lines with the `l` method.

Reply:
There are already a couple of `l` methods in the views:

```
✗ rg -w l src/api/app/views/webui
src/api/app/views/webui/comment/_content.html.haml
8:      = link_to("#comment-#{comment.id}", title: l(comment.created_at.utc), name: "comment-#{comment.id}") do

src/api/app/views/webui/request/_request_history.html.haml
8:      = link_to('#request-creation', title: l(bs_request.created_at.utc), name: 'request-creation') do

src/api/app/views/webui/request/_requests_small.html.haml
15:              %u{ title: l(req.created_at.utc) }

src/api/app/views/webui/request/_request_history_element.html.haml
11:      = link_to("#status-history-#{element.id}", title: l(element.created_at.utc), name: "status-history-#{element.id}") do

src/api/app/views/webui/request/beta_show_tabs/_conversation.html.haml
46:            = link_to('#request-creation', title: l(bs_request.created_at.utc), name: 'request-creation') do

src/api/app/views/webui/request/_show_side_links.html.haml
31:        %span.fuzzy-time{ title: "#{l bs_request.accept_at}" }
33:          = l bs_request.accept_at, format: :only_date
```

For consistency, should we change them also?

Reply:
Good point @eduardoj! This can happen in another PR :slightly_smiling_face: